### DISCUSSION TITLE
Error: Timestamps are unset in a packet for stream 0

### DISCUSSION THREAD

**Discussion by twister36** [blakeblackshear/frigate#9281] [Ask A Question]
I got a new error today on my Lorex Doorbell camera that I don't understand. Hoping that someone will have context on what this means?

"Timestamps are unset in a packet for stream 0. This is deprecated and will stop working in the future. Fix your code to set the timestamps"

![image](https://github.com/blakeblackshear/frigate/assets/114322877/e14cc3d1-3e9a-41e6-ab96-3408a25965ca)

**Comment by NickM-27** [comment]
That's not an error, just a warning. The error is that the camera timed out

**Reply by twister36** [reply]
@NickM-27 - Yes, I see this one camera timeout allot... my other 8 cameras rarely report any issues.
